shell/src/main/java/alluxio/cli/ConfigurationDocGenerator.java

```diff
@@ -164,6 +164,10 @@ static void writeYMLFile(Collection<? extends PropertyKey> defaultKeys, String f
         String description = iteratorPK.getDescription().replace("'", "''");
 
         // Write property key and default value to yml files
+        if (iteratorPK.isIgnoredSiteProperty()) {
+          description = "Note: This property must be specified as a JVM property; "
```

Comment:
the description is more interesting than the warning, so the description should come first

Reply:
Done.